app/experimenter/outcomes/tests/test_outcomes.py

```diff
@@ -0,0 +1,52 @@
+import os
+import pathlib
+
+from django.test import TestCase, override_settings
+
+from experimenter.experiments.constants import NimbusConstants
+from experimenter.outcomes import Outcome, Outcomes
+
+
+@override_settings(
+    JETSTREAM_CONFIG_OUTCOMES_PATH=os.path.join(
```

Comment:
Since we have all the available TOML in the submodule, would it be handy to have a test that iterates through all the current files and parses them for at least a sanity check? More of an integration test than a unit test, but could catch problems early

Reply:
We could do it as a [django system check](https://docs.djangoproject.com/en/3.1/topics/checks/) at startup, that's also a good idea

Reply:
Added a startup check!